R/test-elements.R

```diff
@@ -1557,7 +1563,10 @@ conditional <- function(test, logic) {
                 session = session, opt = opt)
     if (!is.scalar.logical(res)) stop("<test> did not return a ",
                                       "logical scalar")
-    if (!res) skip_n_pages(state, n)
+    if (!res) {
+      message(sprintf("Conditional: Skipping %d pages", n))
```

Comment:
Do you want to keep this in, or can it be removed now?

Reply:
Can be removed